pkg/config/model/viper.go

```diff
@@ -150,6 +151,12 @@ func (c *safeConfig) OnUpdate(callback NotificationReceiver) {
 	c.notificationReceivers = append(c.notificationReceivers, callback)
 }
 
+func getCallerLocation(nbStack int) string {
```

Comment:
Are you considering keeping this function in the long run? Relying on nbStack tends to break when the code evolve.

Reply:
We want to know who called `Set`, is there a better way without relying on callstack ?